packages/contentful--create-contentful-app/src/includeAppAction.ts

```diff
@@ -20,33 +27,30 @@ export async function cloneAppAction(destination: string, templateIsTypescript:
     await d.clone(appActionDirectoryPath);
 
     // move the manifest from the actions folder to the root folder
-    renameSync(
-      `${appActionDirectoryPath}/${CONTENTFUL_APP_MANIFEST}`,
-      `${destination}/${CONTENTFUL_APP_MANIFEST}`
-    );
+    await mergeJsonIntoFile({
+      source: `${appActionDirectoryPath}/${CONTENTFUL_APP_MANIFEST}`,
+      destination: `${destination}/${CONTENTFUL_APP_MANIFEST}`,
+    });
 
     // move the build file from the actions folder to the root folder
-    renameSync(`${appActionDirectoryPath}/build-actions.js`, `${destination}/build-actions.js`);
+    await rename(`${appActionDirectoryPath}/build-actions.js`, `${destination}/build-actions.js`);
```

Comment:
If we're moving to async, we can do this js file and json merge above in parallel :)